CMakeLists.txt

```diff
@@ -75,9 +75,9 @@ else(APPLE)
     find_package(Opus REQUIRED)
     list(APPEND LIBRARIES Opus::opus)
 endif(APPLE)
+find_package(Opus REQUIRED) 
```

Comment:
While you're moving this, it would be good to get rid of the redundant line two lines up.